<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_ProcessClass</class>
 <widget class="QMainWindow" name="Image_ProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image_Process</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_show">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>点运算</string>
    </property>
    <addaction name="actHistm"/>
    <addaction name="actGray"/>
    <addaction name="actLineTrams"/>
    <addaction name="actLog"/>
    <addaction name="actGamma"/>
    <addaction name="actThred"/>
    <addaction name="actSubsection"/>
    <addaction name="actEqualization"/>
    <addaction name="actNorm"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actHistm"/>
   <addaction name="actGray"/>
   <addaction name="actLineTrams"/>
   <addaction name="actLog"/>
   <addaction name="actGamma"/>
   <addaction name="actThred"/>
   <addaction name="actSubsection"/>
   <addaction name="actEqualization"/>
   <addaction name="actNorm"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="Image_Process.qrc">
     <normaloff>:/Images/Resources/loadpic.png</normaloff>:/Images/Resources/loadpic.png</iconset>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="toolTip">
    <string>打开图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="Image_Process.qrc">
     <normaloff>:/Images/Resources/save.png</normaloff>:/Images/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
   <property name="toolTip">
    <string>保存图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actHistm">
   <property name="icon">
    <iconset>
     <normaloff>Resources/histogram.png</normaloff>Resources/histogram.png</iconset>
   </property>
   <property name="text">
    <string>直方图</string>
   </property>
   <property name="toolTip">
    <string>直方图</string>
   </property>
  </action>
  <action name="actLineTrams">
   <property name="icon">
    <iconset>
     <normaloff>Resources/linetransform.png</normaloff>Resources/linetransform.png</iconset>
   </property>
   <property name="text">
    <string>线性变换</string>
   </property>
   <property name="toolTip">
    <string>线性变换</string>
   </property>
  </action>
  <action name="actLog">
   <property name="icon">
    <iconset>
     <normaloff>Resources/logTransform.png</normaloff>Resources/logTransform.png</iconset>
   </property>
   <property name="text">
    <string>对数变换</string>
   </property>
   <property name="toolTip">
    <string>对数变换</string>
   </property>
  </action>
  <action name="actGamma">
   <property name="icon">
    <iconset>
     <normaloff>Resources/gamma.png</normaloff>Resources/gamma.png</iconset>
   </property>
   <property name="text">
    <string>伽马变换</string>
   </property>
   <property name="toolTip">
    <string>伽马变换</string>
   </property>
  </action>
  <action name="actGray">
   <property name="icon">
    <iconset>
     <normaloff>Resources/color_gray.png</normaloff>Resources/color_gray.png</iconset>
   </property>
   <property name="text">
    <string>灰度化</string>
   </property>
   <property name="toolTip">
    <string>灰度化</string>
   </property>
  </action>
  <action name="actEqualization">
   <property name="icon">
    <iconset>
     <normaloff>Resources/equalization.png</normaloff>Resources/equalization.png</iconset>
   </property>
   <property name="text">
    <string>均衡化</string>
   </property>
   <property name="toolTip">
    <string>均衡化</string>
   </property>
  </action>
  <action name="actNorm">
   <property name="icon">
    <iconset>
     <normaloff>Resources/normlization.png</normaloff>Resources/normlization.png</iconset>
   </property>
   <property name="text">
    <string>规定化</string>
   </property>
   <property name="toolTip">
    <string>规定化</string>
   </property>
  </action>
  <action name="actThred">
   <property name="icon">
    <iconset>
     <normaloff>Resources/threshold.png</normaloff>Resources/threshold.png</iconset>
   </property>
   <property name="text">
    <string>阈值分割</string>
   </property>
   <property name="toolTip">
    <string>阈值分割</string>
   </property>
  </action>
  <action name="actSubsection">
   <property name="icon">
    <iconset>
     <normaloff>Resources/subsection.png</normaloff>Resources/subsection.png</iconset>
   </property>
   <property name="text">
    <string>分段线性</string>
   </property>
   <property name="toolTip">
    <string>分段线性变换</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Image_Process.qrc"/>
 </resources>
 <connections/>
</ui>
